core/designsystem/src/commonMain/kotlin/com/droidknights/app/core/designsystem/components/Button.kt

```diff
@@ -1,18 +1,32 @@
 package com.droidknights.app.core.designsystem.components
 
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.semantics.role
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.dp
+import com.droidknights.app.core.designsystem.theme.KnightsTheme
 
+// TODO routing을 위한 임시 구현
 @Composable
 fun Button(
     text: String,
     onClick: () -> Unit,
     modifier: Modifier = Modifier
 ) {
-    androidx.compose.material3.Button(
+    Surface(
         onClick = onClick,
-        modifier = modifier
+        modifier = modifier.semantics { role = Role.Button },
+        shape = RoundedCornerShape(50),
```

Comment:
```suggestion
        shape = RoundedCornerShape(percent = 50),
```

이렇게 해야 조금 더 직관적으로 어떤 의미인지 와닿을 것 같아요

Reply:
@l2hyunwoo 좋아요. 그런데 이건 별개의 얘기인데 RoundCornerShape(50) == CircleShape로 값이 같은데 이럴때 CircleShape를 쓰면 또 헷갈리기도 하겠네용